pages/bare_metal_cloud/virtual_private_servers/starting_with_a_vps/guide.fr-fr.md

```diff
@@ -145,29 +161,39 @@ Vous pouvez également sélectionner une **clé SSH** à installer sur le systè
 > Pour avoir un système d’exploitation **Windows** fonctionnant sur un VPS, il faut l'avoir préalablement choisi **dans le processus de commande**. Un VPS avec un autre OS installé ne peut pas être réinstallé avec Windows via la méthode décrite ci-dessus.
 >
 
-Une barre de progression de l'installation s'affiche dans votre espace client. Veuillez noter que ce processus peut prendre jusqu'à 30 minutes.
+Le processus de réinstallation peut prendre quelques minutes.
 
-#### Redémarrage de votre VPS <a name="reboot-current-range"></a>
+> [!warning]
+>
+> Pour des raisons de sécurité, lors de la première connexion à votre VPS, vous devez changer le mot de passe reçu par mail pour le remplacer par un mot de passe personnel et fort. Une fois la modification effectuée, reconnectez-vous avec votre nouveau mot de passe.
+>
+
+#### Redémarrer votre VPS <a name="reboot-current-range"></a>
 
 Un redémarrage peut s'avérer nécessaire afin d'appliquer des configurations de mises à jour ou pour résoudre un dysfonctionnement. Dans la mesure du possible, effectuez un « redémarrage logiciel » à partir de l'interface graphique du serveur (Windows, Plesk, etc.) ou via la ligne de commande :
 
 ```bash
 sudo reboot
 ```
 
-Cependant, vous pouvez effectuer un « redémarrage matériel » à tout moment dans votre [espace client OVHcloud](/links/manager). Depuis l'onglet `Accueil`{.action}, cliquez sur le bouton `...`{.action} à côté de `Boot` dans la section **Votre VPS**. Sélectionnez `Redémarrer mon VPS`{.action} et cliquez sur `Valider`{.action} dans la fenêtre qui s'affiche.
+Cependant, vous pouvez effectuer un redémarrage forcé à tout moment dans votre [espace client OVHcloud](/links/manager). Depuis l'onglet `Accueil`{.action}, cliquez sur le bouton `...`{.action} à côté de `Boot` dans la section **Votre VPS**. Sélectionnez `Redémarrer mon VPS`{.action} et cliquez sur `Valider`{.action} dans la fenêtre qui s'affiche.
 
 ![Reboot](images/reboot-vps01.png){.thumbnail}
 
 <a name="connect"></a>
 
-### Connexion à votre VPS (OS GNU/Linux)
+### Se connecter à votre VPS
 
-Lors de la première installation ou lors de la réinstallation à partir du Panneau de configuration, un utilisateur avec des autorisations élevées est créé automatiquement. Cet utilisateur sera nommé en fonction du système d'exploitation, par exemple « ubuntu » ou « rocky ».
+> [!warning]
+>
+> Pour des raisons de sécurité, lors de la première connexion à votre VPS, vous devez changer le mot de passe reçu par mail pour le remplacer par un mot de passe personnel et fort. Une fois la modification effectuée, il est possible que l'interface que vous utilisez (Putty par exemple) se ferme automatiquement afin de procéder à la déconnexion. Reconnectez-vous avec votre nouveau mot de passe.
```

Comment:
```suggestion
> Pour des raisons de sécurité, lors de la première connexion à votre VPS, vous devez changer le mot de passe reçu par mail pour le remplacer par un nouveau mot de passe fort. Une fois la modification effectuée, il est possible que l'interface que vous utilisez (Putty par exemple) se ferme automatiquement afin de procéder à la déconnexion. Reconnectez-vous alors avec votre nouveau mot de passe.
```
Same remark as above regarding the mandatory aspect of this password change.

Reply:
Yes the UI forces the user to change his password